build.spec

```diff
@@ -76,6 +76,12 @@ a = Analysis(
     win_private_assemblies=False,
     win_no_prefer_redirects=False,
 )
+# Exclude unneeded Linux libraries
+a.binaries = a.binaries - TOC([
+  ('libicudata.so.66', None, None),
+  ('libicuuc.so.66', None, None),
+  ('librsvg-2.so.2', None, None),
+])
```

Comment:
`a.binaries` is just a list of 3-item tuples, so this would be way to go:

```suggestion

# Exclude unneeded Linux libraries
excluded_binaries = [
    "libicudata.so.66",
    "libicuuc.so.66",
    "librsvg-2.so.2"
]
a.binaries = [b for b in a.binaries if b[0] not in excluded_binaries]

```